docs/development.md

```diff
@@ -0,0 +1,23 @@
+# Development guideline
+
+## Environment setup
+Here is how to setup the development environment:
+1. Setup the project by first installing the dependencies `npm install`
+2. Write code and tests
+3. Make sure all tests pass `npm test`
+4. Make sure code is well formatted and secure `npm run lint`
+
+## CI Blackbox testing
+In each PR there will be running a CI pipeline which ensures that this [dummy.json](../test/CI-blackbox/dummy.yml) file can produce a valid output file. This means all generated models will be written to files and then transpiled/compiled depending on the output language. 
+
+If any of those CI jobs fail this is how to reproduce the output.
+
+For each output language (`ts`, `js`, `java`) there will be the following scripts:
+1. `ci:blackbox:<language>:generate` which generates the output
```

Comment:
I think you need to update this

Reply:
Probably `CI Blackbox testing` and `Requirements` should be updated.

Reply:
I completely forgot about the docs, sorry 😅 

I have changed it.